Messenger/AppDelegate.mm

```diff
@@ -186,6 +186,7 @@ - (void)applicationDidFinishLaunching:(NSNotification*)notification {
   #if 0
   webView.customUserAgent = @"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/600.7.12 (KHTML, like Gecko) Version/8.0.7 Safari/600.7.12";
   #endif // 0
+  webView.maintainsBackForwardList = NO;
```

Comment:
This removes the “Forward“ and “Back“ actions from the menu as well, since those seem to just cause problems in this app. However, maybe you still want them to show up in debug mode?